scout/server/blueprints/panels/templates/panels/gene-edit.html

```diff
@@ -44,7 +44,7 @@
             </div>
             <div class="col-md-4">
               {{ form.custom_inheritance_models.label(class="control-label") }}
-              {{ form.custom_inheritance_models(class="form-control", placeholder="free text, comma separated") }}
+              {{ form.custom_inheritance_models(class="form-control", placeholder="Valid models are: AR, AD, MT, XD, XR, X, Y") }}
```

Comment:
They still need to be the usual models I think. And there is a check downstream anyway. The fields where checked also in the original cose, see [here](https://github.com/Clinical-Genomics/scout/blob/6fa569171523265b5b2f8d7467d1bdfb4bc6e3d6/scout/build/panel.py#L85)

Reply:
Gotcha, right, I remembered there was something off about it. I agree the behaviour should be the same between the page input and the loading from file! Do you have a strong opinion already, or should we check if there is still a lot of "custom" models around the prod panels?

Reply:
The check you link puts anything that *doesn't* match the regular models in custom, right?

Reply:
Is the field even used downstream? Because if it isn't one could write whatever, I guess. But then how useful is it? 🤔

Reply:
> The check you link puts anything that _doesn't_ match the regular models in custom, right?

Nope, allows in custom only the standard models.

Reply:
I read it as if NOT in valid_models, append to custom models.

Reply:
> Is the field even used downstream? Because if it isn't one could write whatever, I guess. But then how useful is it? 🤔

A quick browse says it is rare, but I found e.g. a few with inheritance model "DR" for digenic, where the custom field is in use.

Reply:
Ah perhaps I can revert the text in this placeholder